src/components/CropSelector/CropTable.js

```diff
@@ -199,143 +200,141 @@ const CropTableComponent = (props) => {
   };
 
   const CropList = ({ matchGoals }) => {
-    return activeCropPresent
-      ? activeCropData.map((crop, index) => {
-          if (
-            crop.fields['Zone Decision'] === 'Include' &&
-            (matchGoals ? !hasGoalRatingTwoOrLess(crop) : hasGoalRatingTwoOrLess(crop))
-          )
-            return (
-              <Fragment key={index}>
-                <TableRow
-                  className={hasGoalRatingTwoOrLess(crop) ? `inactiveCropRow` : ''}
-                  key={`croprow${index}`}
-                  id={crop.fields['id']}
-                  style={hasGoalRatingTwoOrLess(crop) ? { opacity: '0.2' } : {}}
-                >
-                  <TableCell style={{ height: 'auto' }}>
-                    <div className="container-fluid">
-                      <div className="row">
-                        <div className="col-auto pl-md-0">
-                          {crop.fields['Image Data'] ? (
-                            <CropImage
-                              present={true}
-                              src={
-                                crop.fields['Image Data']['Key Thumbnail']
-                                  ? `/images/Cover Crop Photos/100x100/${crop.fields['Image Data']['Directory']}/${crop.fields['Image Data']['Key Thumbnail']}`
-                                  : 'https://placehold.it/100x100'
-                              }
-                              alt={crop.fields['Cover Crop Name']}
-                            />
-                          ) : (
-                            <CropImage present={false} />
-                          )}
+    return (
+      activeCropPresent &&
+      activeCropData.map((crop, index) => {
+        if (
+          crop.fields['Zone Decision'] === 'Include' &&
+          (matchGoals ? !hasGoalRatingTwoOrLess(crop) : hasGoalRatingTwoOrLess(crop))
+        )
+          return (
+            <Fragment key={index}>
+              <TableRow
+                className={hasGoalRatingTwoOrLess(crop) ? `inactiveCropRow` : ''}
+                key={`croprow${index}`}
+                id={crop.fields['id']}
+                style={hasGoalRatingTwoOrLess(crop) ? { opacity: '0.2' } : {}}
+              >
+                <TableCell style={{ height: 'auto' }}>
+                  <div className="container-fluid">
+                    <div className="row">
+                      <div className="col-auto pl-md-0">
+                        {crop.fields['Image Data'] ? (
```

Comment:
Im wondering if this will still work if we remove the initial ternary and only apply the ternary to the present prop the nested ternary is ok here and could probably be adjusted to work.  Not sure on initial review.